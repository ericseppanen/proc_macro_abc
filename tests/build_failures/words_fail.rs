use abc_macros::file_words;

fn main() {
    let _ = file_words!("tests/words/nonexistent");
}
