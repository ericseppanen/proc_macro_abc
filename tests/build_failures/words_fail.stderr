error: Failed to read file
 --> $DIR/words_fail.rs:4:25
  |
4 |     let _ = file_words!("tests/words/nonexistent");
  |                         ^^^^^^^^^^^^^^^^^^^^^^^^^
